packages/nc-gui/components/account/UserList.vue

```diff
@@ -59,7 +61,7 @@ const updateRole = async (userId: string, roles: Role) => {
   try {
     await api.orgUsers.update(userId, {
       roles,
-    } as unknown as UserType)
+    } as OrgUserReqType)
```

Comment:
We can even remove the `as OrgUserReqType`

Reply:
Type 'Role' is not assignable to type '"org-level-creator" | "org-level-viewer" | undefined'.